content/docs/lists-and-keys.md

```diff
@@ -94,11 +94,11 @@ ReactDOM.render(
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/jrXYRR?editors=0011)
+[**Experimente no CodePen**](https://codepen.io/gaearon/pen/jrXYRR?editors=0011)
 
-## Keys {#keys}
+## Chaves {#keys}
```

Comment:
I think it's good to stick with `Keys` as it actually means a named attribute (`key`)